pkg/inventory/web/web.go

```diff
@@ -13,6 +15,8 @@ const (
 	NsParam      = "ns1"
 	NsCollection = "namespaces"
 	Root         = "/" + NsCollection + "/:" + NsParam
+	TlsCert      = "/var/run/secrets/inventory-tls/tls.crt"
+	TlsKey   = "/var/run/secrets/inventory-tls/tls.key"
```

Comment:
Lets add fields to the WebServer (like):
```
type WebServer struct {
    ...
    // TLS
   TLS struct {
        // Certificate path.
        Certificate string
        // Key path.
        Key string
   }
```

Reply:
In the virt-controller in would image something like:
In the settings package: inventory.go
```
const (
     ...
     API_TLS_CERTIFICATE
     API_TLS_KEY
)

...

type inventory struct {
    ...
    // TLS settings.
   TLS struct {
        // Certificate path.
        Certificate string
        // Key path.
        Key string
   }
}
```
Set the TLS fields in Load().

In the controller (Example virt provider):

```
  web.Port = Settings.Inventory.Port
+ web.TLS.Certificate = Settings.Inventory.TLS.Certificate
+ web.TLS.Key = Settings.Inventory.TLS.Key
```

Reply:
The virt-controller [client](https://github.com/konveyor/virt-controller/blob/master/pkg/controller/provider/web/base/client.go#L99) will need to be updated:
The `url()` method could use the Settings.Inventory.Scheme().
The `get()` method could use the Settings.Inventory.TLSEnabled() to determine how to build the `http` client.